### PR TITLE
EQL: Disable field extraction for source return

### PR DESCRIPTION
**Pull request by costin**
Metric mappings have lots of mappings which, in case of individual field selection, can easily hit the limit of maximum doc value fields or automatons.

As fields are not used in EQL, this commit disables field extraction and returns the entire source instead.

### CODE REVIEW

**Review by aleksmaus — APPROVED** [review]

File: x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/execution/search/SourceGenerator.java [hunk 1]
```diff
@@ -45,10 +45,7 @@ public static SearchSourceBuilder sourceBuilder(QueryContainer container, QueryB
         // Iterate through all the columns requested, collecting the fields that
         // need to be retrieved from the result documents
 
-        // NB: the sortBuilder takes care of eliminating duplicates
-        container.fields().forEach(f -> f.v1().collectFields(sortBuilder));
-        sortBuilder.build(source);
```
Comment (astefan) [inline x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/execution/search/SourceGenerator.java]: The `sortBuilder` is still defined above. Needs removal? Also, the two comments above are not needed anymore.

File: client/rest-high-level/src/test/java/org/elasticsearch/client/EqlIT.java [hunk 2]
```diff
@@ -61,4 +63,31 @@ public void testBasicSearch() throws Exception {
...
         assertThat(response.hits().events().size(), equalTo(1));
     }
+
+    public void testLargeMapping() throws Exception {
+        Request doc1 = new Request(HttpPut.METHOD_NAME, "/index/_doc/1");
+
+        String now = DateUtils.nowWithMillisResolution().format(DateTimeFormatter.ISO_DATE_TIME);
+        StringBuilder sb = new StringBuilder();
+        sb.append("{");
+        for (int i = 0; i < 250; i++) {
```
Comment (astefan) [inline client/rest-high-level/src/test/java/org/elasticsearch/client/EqlIT.java]: I would add a comment here explaining the reason for 250 value (that value larger than the 100 doc_values limit in ES). Maybe take the default limit in ES and use here a value made of that limit + 100 or something like that?

**Review by astefan — APPROVED** [review]
LGTM. Left two minor comments.

**Comment by costin** [comment]
@elasticmachine run elasticsearch-ci/bwc

**Pull request merged**